Wikipedia/Code/ViewController.swift

```diff
@@ -7,6 +7,10 @@ class ViewController: PreviewingViewController, Themeable, NavigationBarHiderDel
     init() {
         super.init(nibName: nil, bundle: nil)
     }
+
+    override init(nibName nibNameOrNil: String?, bundle nibBundleOrNil: Bundle?) {
```

Comment:
You can delete this.

Reply:
I'm using that init in `EditLinkViewController`, without it I'll get an `Ambiguous reference to member 'init()'` error

Reply:
ah ok my mistake, just thought it was a missed unused override